platforms/ios/demo/src/MapViewController.m

```diff
@@ -18,6 +18,15 @@ - (void)addAlert:(NSString *)message withTitle:(NSString *)title;
 
 @implementation MapViewControllerDelegate
 
+- (void)mapView:(TGMapViewController *)mapView didCompleteSceneUpdatesWithErrors:(NSArray<TGSceneUpdateStatus *> *)updateStatuses
+{
+    for (TGSceneUpdateStatus* updateStatus in updateStatuses) {
+        NSLog(@"Scene update error for update with path %@ and value %@",
```

Comment:
As I mentioned for android interface, instead of logging, does it make sense to send a NSError here? cc. @msmollin